packages/asc-ui/src/components/Link/LinkStyle.ts

```diff
@@ -1,18 +1,74 @@
-import styled, { css } from '@datapunt/asc-core'
+import styled, { css, Theme } from '@datapunt/asc-core'
 import { color } from '../../utils'
-import { focusStyleText } from '../../utils/themeUtils'
-import Typography from '../Typography'
+import { focusStyleText, svgFill } from '../../utils/themeUtils'
+import Typography, { TypographyProps } from '../Typography'
+import IconStyle from '../Icon/IconStyle'
 
-export const LinkStyleCSS = css`
-  color: ${color('primary')};
+export const BlankLinkStyleCSS = css`
   display: inline-block;
-  ${focusStyleText()}
+  text-decoration: none;
+  color: inherit;
+`
+export const InlineLinkStyleCSS = ({
+  theme,
+}: {
+  theme: Theme.ThemeInterface
+}) => css`
+  display: inline-block;
+  color: ${color('primary')({ theme })};
 
   &:hover {
     color: ${color('secondary')};
   }
 `
+export const DefaultLinkStyleCSS = ({
+  theme,
+}: {
+  theme: Theme.ThemeInterface
+}) => css`
+  display: flex;
+  color: ${color('tint', 'level6')({ theme })};
+  text-decoration: none;
+  font-weight: 700;
+
+  ${IconStyle} {
+    margin: 3px;
+  }
+
+  &:hover {
+    color: ${color('secondary')({ theme })};
+    text-decoration: underline;
+    ${IconStyle} {
+      ${svgFill('secondary')({ theme })};
+    }
+  }
+`
 
-export default styled(Typography)`
-  ${LinkStyleCSS}
+type TypeLink = 'default' | 'inline' | 'blank' | 'default-chevron'
```

Comment:
Hmmm I think it might be better to leave the 'default' as a prop (Like if the user doesn't pass a prop, it will fallback to the default style. Then you can just fallback to the default style in the switch statement below. Perhaps name the default-chevron -> withChevron?

Reply:
done